src/pages/Beers.jsx

```diff
@@ -0,0 +1,50 @@
+import { useState, useEffect, useContext } from "react";
+import axios from "axios";
+// import { ThemeContext } from "../contexts/theme";
+import { Link } from "react-router-dom";
+import { useParams } from 'react-router-dom';
+
+
+import React from "react";
+
+function Beers() {
+  const [beers, setBeers] = useState([]);
+  const { beerId } = useParams();
+  const getBeers = async () => {
+    try {
+      const apiCall = await axios.get(
+        `https://ih-beers-api2.herokuapp.com/beers/${beerId}`
```

Comment:
The correct link to get all the beers from the api is: `'https://ih-beers-api2.herokuapp.com/beers'`